LCM/dsc/engine/EngineHelper/EventWrapper.c

```diff
@@ -25,8 +25,10 @@ ConfigurationDetails g_ConfigurationDetails = { 0, 0, MI_FALSE };
 #else
 #include <pal/cpu.h>
 ConfigurationDetails g_ConfigurationDetails;
-static FILE *_DSCos;
-static Log_Level _DSClevel = OMI_WARNING;
+static FILE *_DSCLogFile;
+static FILE *_DSCDetailedLogFile;
+static Log_Level _DSCLogLevel = OMI_WARNING;
+static Log_Level _DSCDetailedLogLevel = OMI_VERBOSE;
```

Comment:
Currently we are not getting any messages about DSC engine activities in DSC log file. 
All the engine logs are INFO level logs. 
Idea is to add those logs somewhere. we cannot write INFO level logs in dsc.log/omsconfig.log file because of the size. so with this change, Engine will create two log files. 1. omsconfig.log/dsc.log it will have error/warning level information, and we will maintain longer log history in this file. dscdetailed.log/omsconfigdetailed.log file will have all the logs including INFO level information. it will keep 3-4 days log history.

---
In reply to: [140115027](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140115027) [](ancestors = 140115027)

Reply:
ok got it. thanks for explaining. 

---
In reply to: [140366875](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140366875) [](ancestors = 140366875,140115027)